{"name": "test_unsuccessfull_login_courier_without_login_field", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Недостаточно...ных для входа' == 'Недостаточно...четной записи'\n  \n  \u001b[0m\u001b[91m- Недостаточно данных для создания учетной записи\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n  \u001b[92m+ Недостаточно данных для входа\u001b[39;49;00m\u001b[90m\u001b[39;49;00m", "trace": "self = <tests.test_login_courier.TestLoginCourier object at 0x000001A3D0A24050>, сreate_courier = ['usdlykwdlh', 'bsvhhvdnhb', 'pbmqagrkjx']\n\n    def test_unsuccessfull_login_courier_without_login_field(self, сreate_courier):\n            with allure.step(\"Формирование тела запроса\"):\n                payload = {\n                    \"password\": сreate_courier[1]\n                }\n    \n            with allure.step(\"отправляем запрос на регистрацию курьера и сохраняем ответ в переменную response\"):\n                response = requests.post('https://qa-scooter.praktikum-services.ru/api/v1/courier/login', data=payload)\n            with allure.step(\"проверка кода и тела ответа\"):\n                assert response.status_code == 400\n>               assert response.json()[\"message\"] == \"Недостаточно данных для создания учетной записи\"\nE               AssertionError: assert 'Недостаточно...ных для входа' == 'Недостаточно...четной записи'\nE                 \nE                 \u001b[0m\u001b[91m- Недостаточно данных для создания учетной записи\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\nE                 \u001b[92m+ Недостаточно данных для входа\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n\ntests\\test_login_courier.py:45: AssertionError"}, "steps": [{"name": "Формирование тела запроса", "status": "passed", "start": 1757334972981, "stop": 1757334972981}, {"name": "отправляем запрос на регистрацию курьера и сохраняем ответ в переменную response", "status": "passed", "start": 1757334972981, "stop": 1757334973287}, {"name": "проверка кода и тела ответа", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Недостаточно...ных для входа' == 'Недостаточно...четной записи'\n  \n  \u001b[0m\u001b[91m- Недостаточно данных для создания учетной записи\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n  \u001b[92m+ Недостаточно данных для входа\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n", "trace": "  File \"c:\\Users\\Dmitry\\Desktop\\Python YP\\Sprint_7\\tests\\test_login_courier.py\", line 45, in test_unsuccessfull_login_courier_without_login_field\n    assert response.json()[\"message\"] == \"Недостаточно данных для создания учетной записи\"\n"}, "start": 1757334973287, "stop": 1757334973287}], "start": 1757334972981, "stop": 1757334973288, "uuid": "69171184-d38e-4f1e-b2bb-83526e00b402", "historyId": "cb98a2696d0456f1475a74264a44f2ff", "testCaseId": "cb98a2696d0456f1475a74264a44f2ff", "fullName": "tests.test_login_courier.TestLoginCourier#test_unsuccessfull_login_courier_without_login_field", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-A445VSF"}, {"name": "thread", "value": "13284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}], "titlePath": ["tests", "test_login_courier.py", "TestLoginCourier"]}